### ISSUE TITLE
selected option from Select component doesn't mark the option as selected

### ISSUE THREAD

**Issue by EmmaNaty** [driftyco/ionic#7695]
I recently updated my project to work with Ionic 2 Beta 11 so I changed the code of the select component 

from
`<ion-item>
  <ion-label>{{abel_}}</ion-label>
  <ion-select [(ngModel)]="selected_value" >
    <ion-option *ngFor="let a of data_provider.list" value="{{a.id}}"  checked="{{a.checked}}">
        {{a.label}}
    </ion-option>
  </ion-select>
</ion-item>
`

to
`<ion-item>
  <ion-label>{{abel_}}</ion-label>
  <ion-select [(ngModel)]="selected_value" >
    <ion-option *ngFor="let a of data_provider.list" value="{{a.id}}"  selected="{{a.checked}}">
        {{a.label}}
    </ion-option>
  </ion-select>
</ion-item>
`

But when I use the option **selected** instead of **checked** the option doesn't mark as selected.

**Comment by manuelroth** [comment]
Have the same problem.

**Comment by manuelroth** [comment]
@EmmaNaty A workaround which worked for me, was to set the selected_value in the constructor of my component:

```
  constructor() {
    this.selected_value = "OPTION_WHICH_SHOULD_BE_SELECTED_PER_DEFAULT";
  }
```

**Comment by almothafar** [comment]
The same issue:
```
        <ion-select [(ngModel)]="selectedOption" mode="md" (ionChange)="selectOptionChange($event)">
          <ion-option *ngFor="let opt of options" [value]="opt.value">{{opt.label}}</ion-option>
        </ion-select>
```

And:

```
  constructor(private cdRef: ChangeDetectorRef) {
  }

  ngOnInit(): void {
    this.cdRef.detectChanges();
    if (!!this.options) {
      this.selectOptionChange(this.options[0].value);
    }

  }

  ngOnDestroy(): void {
    this.cdRef.detach();
  }

  selectOptionChange(option) {
    this.selectedOption = option.value;
    // console.debug(option);
    this.optionChange.emit(option);
  }
```
